Makefile

```diff
@@ -24,24 +24,24 @@ build_race:
 dist:
 	@BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/dist.sh'"
 
-test:
-	@echo "--> Running linter"
-	@make metalinter_test
-	@echo "--> Running go test"
-	@go test $(PACKAGES)
+test: tools get_vendor_deps
```

Comment:
no, test should not invoke tools or get_vendor_deps. they are both too time consuming! and dev time is damn expensive. Instead, we should add a section in readme (if we don't have it already):
>>>>>>>>>>>>>

## Development

First two command only needed to be executed once. If you are using Vagrant, just go straight to testing (`make test`).

```
make get_vendor_deps
make tools
make test
```